javagen/src/main/java/com/azure/autorest/template/ClientMethodTemplate.java

```diff
@@ -1578,4 +1580,15 @@ protected void generateSendRequestSync(ClientMethod clientMethod, JavaType typeB
             function.methodReturn("this.sendRequestAsync(httpRequest).contextWrite(c -> c.putAll(FluxUtil.toReactorContext(context).readOnly())).block()");
         });
     }
+
+    private String getServiceVersionValue(ClientMethod clientMethod) {
```

Comment:
can it be static?

Reply:
It can be static. But I don't quite get the benefit of making it static? The method is `private` , so no instance should be instantiated to call it.

Reply:
`static` means it will never use any var in the class. So whenever method can be static, it better be.

Reply:
Make sense, I will make it static.